v3/plugins/redhat/jaeger/1.14/meta.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v2
+publisher: redhat
+name: jaeger
+version: 1.14
+type: Che Plugin
+displayName: Jaeger
+title: Jaeger, Distributed Tracing System
+description: OpenTracing/OpenTelemetry compliant distributed tracing plugin for Eclipse Che
+icon: https://raw.githubusercontent.com/jaegertracing/documentation/master/static/img/jaeger-vector.svg
+category: Observability
```

Comment:
Wasn't sure whether to set this to `Other`, but thought might be good to have a new category for observability tooling.

Reply:
Have changed to `Other` due to build failure - but might be worth considering a new category :)